packages/dev/gui/src/2D/controls/gradient/BaseGradient.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Base Gradient class. Should not be used directly.
+ */
+
+import type { ICanvasGradient, ICanvasRenderingContext } from "core/Engines/ICanvas";
+
+/**
+ * Type that represents a single stop on the gradient.
+ */
+export type ColorStop = { offset: number; color: string };

```

Comment:
In UMD this will be available in the main namespace.  A more specific name would make sense when used in UMD (`GradientColorStop` maybe?)